### PR TITLE
[SMALLFIX] Fix javadoc errors

### PR DESCRIPTION
**Pull request by aaudiber**
persistFile is no longer done by the worker, so we can remove the feature
from the worker package javadoc

### CODE REVIEW

**Comment by calvinjia** [comment]
LGTM

File: servers/src/main/java/tachyon/worker/package-info.java [hunk 1]
```diff
@@ -13,14 +13,6 @@
  * The thrift service is mostly responsible for metadata operations (with a few operations that
  * effect the worker's cached memory).
  *
- * <h3>Checkpoint</h3>
```
Comment (jsimsa) [inline servers/src/main/java/tachyon/worker/package-info.java]: why is this removed?

Comment (aaudiber) [inline servers/src/main/java/tachyon/worker/package-info.java]: we removed this functionality from the worker with https://github.com/amplab/tachyon/pull/1890, right?

Comment (jsimsa) [inline servers/src/main/java/tachyon/worker/package-info.java]: ah, yes
